BookPlayer/Utils/BaseViewController.swift

```diff
@@ -17,14 +18,45 @@ class BaseViewController<T: Coordinator, U: BaseViewModel<T>>: UIViewController
   }
 }
 
+protocol ViewControllerProtocol: UIViewController {
+  associatedtype VM: ViewModelProtocol
+
+  var viewModel: VM! { get set }
+}
+
+extension ViewControllerProtocol {
+  func accessibilityPerformEscape() -> Bool {
+    self.viewModel.dismiss()
+    return true
+  }
+}
+
+@available(*, deprecated, message: "Use `TableViewControllerProtocol` instead.")
```

Comment:
I think we there are some improvements we can do based on this file:
- Rename `ViewControllerProtocol` to `MVVMControllerProtocol`
- Delete `BaseTableViewController`
- Delete `TableViewControllerProtocol`, and use instead the renamed `MVVMControllerProtocol` (the missing `accessibilityPerformEscape` was an oversight on my part)

Reply:
I just pushed the new code, I hope it's ok. 😁